README.md

```diff
@@ -1,2 +1,2 @@
-# authmodule
+# authmodule, Ayush Malik
```

Comment:
Let's remove the name.

Reply:
Let's resolve the review comments with proper description once done.

Reply:
Name has been removed.